lib/Runtime/Language/InterpreterProcessOpCodeAsmJs.h

```diff
@@ -122,6 +122,15 @@
 #define PROCESS_D2toD1Mem(name, func) PROCESS_D2toD1Mem_COMMON(name, func,)
 #define PROCESS_F2toF1Mem(name, func) PROCESS_F2toF1Mem_COMMON(name, func,)
 
+#define PROCESS_EMPTYASMJS(name, func) \
```

Comment:
Alright, I didn't think about the argument.
You can keep `PROCESS_EMPTYASMJS`, but what about `ip = m_reader.GetIP()` ?